### ISSUE TITLE
missing figures when running marp with --pdf

### ISSUE THREAD

**Issue by tosaba** [yhatt/marp#242]
:arrow_right:  This issue is related to the new version (https://github.com/marp-team/marp-cli), but due to the lack of the _issues_ option, I'm posting it here ...

I have been testing the marp-cli and also in that early stage it's already a very nice and promising tool.
There is one thing I came across when I run the client version with PDF option., i.e.

``` bash
marp test.md --pdf 
```

Local figures (inline or background), which are linked with absolute or relative paths, are not shown in the PDF. 

``` markdown
---
# New slide title

![](figure.png)

![bg](figure.png)
```

Inserting figures from the web works well, e.g.

``` markdown
---

# 1st slide with inline image

![](https://c2.staticflickr.com/2/1688/24269660074_b675f702d7_h.jpg)

---

# 2nd slide with background image

![bg](https://c2.staticflickr.com/2/1688/24269660074_b675f702d7_h.jpg)

---
```

In any case, the HTML output is just fine, no matter if local figures or online figures are being used.

Thanks

**Comment by yhatt** [comment]
Thanks you for testing the early Marp. We have not yet decided the strategy for tracking issues about Marp family. But it's ok to use yhatt/marp for a while.

Perhaps it comes from rendering HTML data by using URI scheme while converting into PDF.

https://github.com/marp-team/marp-cli/blob/bd6d29eb4f67838ae312bd59fc787f4a5069efb5/src/converter.ts#L69-L76

In this case, it has not defined base path to resolve relative path. We are going to try the [HTML `<base>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) that might be helpful..

**Comment by yhatt** [comment]
We're trying to fix in marp-team/marp-cli#10 now. For security reasons, we would have to add `--allow-local-files` option to render local files.

We will let you know once it's released.

**Comment by yhatt** [comment]
@tosaba We have released [`@marp-team/marp-cli` v0.0.6](https://www.npmjs.com/package/@marp-team/marp-cli) it includes fix about reported issue. You would be able to render local figures in PDF correctly by passing `--allow-local-files` option.

**Comment by tosaba** [comment]
Thanks a lot!
